languages/native.ini

```diff
@@ -22,6 +22,7 @@ Hindi = "हिंदी"
 Irish = "Gaeilge"
 Italian = "Italiano"
 Japanese = "日本語"
+Maori = "Māori"
```

Comment:
Note that I tried to use "Māori" everywhere instead of "Maori" but the special character caused some problems, so I ended up with "Maori" in config.ini, and this translation to apply the appropriate accent when displaying it in the UI. Not ideal, but I think it should be okay.

Reply:
Thanks @demiankatz, to be correct we should probably use Maaori if it's impossible to add the special character.